components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/cache/SessionContextCache.java

```diff
@@ -54,6 +54,11 @@ public static SessionContextCache getInstance() {
     }
 
     public void addToCache(SessionContextCacheKey key, SessionContextCacheEntry entry) {
+
+        if(log.isDebugEnabled()) {
+            log.debug("Adding session context corresponding to the key : " + key.getContextId() +
+                " with accessed time " + entry.getAccessedTime() + " and validity time " + entry.getValidityPeriod());
+        }
```

Comment:
Would there be any occasion, some code trying to put a null entry against the key? Then there is a possible NPE here.

Reply:
No. the key will be always not be null